update_ip/ip_getters/base.py

```diff
@@ -1,42 +1,49 @@
 import re
-import urllib2
 import socket
+import urllib2
 
 socket.setdefaulttimeout(5)
 
+
 class GetIpFailed(Exception):
     pass
 
-ip_regex= re.compile("\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}")
+
+# Check for valid IP Addresses
+ip_regex = re.compile(r"(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)")
```

Comment:
You can test both regex online in regex101.com that is what I use. Select python from sidebar. And you can test multiple IPS (valid and not valid) it's not a complicated regex but if need more explanation this web can explain each part.